src/lib/stitching/exchange/stitching.ts

```diff
@@ -351,6 +352,90 @@ export const exchangeStitchingEnvironment = ({
           },
         },
       },
+      Mutation: {
+        // monkey-patch the existing exchange mutation
+        // to add operations dependent on impulse
+        commerceCreateInquiryOfferOrderWithArtwork: {
```

Comment:
question @starsirius: here we are basically re-using the exchange name. Is this name good or you have other suggestions?

Reply:
I see this follows the pattern so I'm 👍 !